packages/ExternalLink/src/ExternalLink.styles.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import { truncateText } from "@paprika/stylers/lib/includes";
+import { toInt } from "@paprika/stylers/lib/helpers";
+
+export const ExternalLinkStyled = styled.a`
+  border-radius: ${tokens.border.radius};
+  color: ${tokens.textColor.link};
+  display: inline-block;
+  position: relative;
+
+  ${props => {
+    return `padding: 1px ${props.externalLinkIconSize + toInt(tokens.spaceSm)}px 1px ${tokens.spaceSm};`;
+  }}
+
+  &:focus,
+  &:active {
+    box-shadow: ${tokens.highlight.active.boxShadow};
+    outline: none;
+  }
+`;
+
+const externalLinkContentHoverFocus = `
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+`;
+
+export const ExternalLinkContentStyled = styled.span`
+  display: inline-block;
+  ${truncateText}
+  vertical-align: bottom;
+  width: 100%;
+
+  ${props => {
+    return props.hasUnderline ? "text-decoration: underline" : externalLinkContentHoverFocus;
+  }};
+`;
+
+export const ExternalLinkIconStyles = `
+  display: inline-block;
+  margin-left: ${tokens.spaceSm};
+  position: absolute;
```

Comment:
Had to make this icon `position:absolute` as was seeing browser differences in how the component was evaluating the width and when it was applying the `text-overflow: ellipses`. Specifically Safari was showing differences. Keeping`postion:absolute` seems to make more consistent between browsers.